org.jhotdraw8.fxbase/src/test/java/org.jhotdraw8.fxbase/org/jhotdraw8/fxbase/undo/UndoableTextFilterTest.java

```diff
@@ -17,7 +17,7 @@ public void shouldFireChangeEvents() {
         Platform.startup(() -> {
         });
 
-        PlatformUtil.invokeAndWait(1000, () -> {
+        PlatformUtil.invokeAndWait(10000, () -> {
```

Comment:
I increased the timeout for this test. It failed consistently when run via the action.